cmd/gh-slack/cmd/auth.go

```diff
@@ -0,0 +1,79 @@
+package cmd
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/cli/go-gh/pkg/config"
+	"github.com/rneatherway/slack"
+	"github.com/spf13/cobra"
+)
+
+var authCmd = &cobra.Command{
+	Use:   "auth [flags]",
+	Short: "Prints authentication information for the Slack API",
+	Long:  `Prints authentication information for the Slack API.`,
```

Comment:
Maybe a simple update like this would be enough?
```suggestion
	Short: "Prints authentication information for the Slack API (treat those as secrets)",
	Long:  `Prints authentication information for the Slack API (treat those as secrets).`,
```